### ISSUE TITLE
Environment Variables appear to be broken in 1.38

### ISSUE THREAD

**Issue by kcharwood** [fastlane/fastlane#819]
All of my lanes worked fine with 1.37. With 1.38, none of them are loaded.

Doing something like this: 

`fastlane test_framework --env ios91`

.env:
```
AF_WORKSPACE="AFNetworking.xcworkspace"

AF_IOS_FRAMEWORK_SCHEME="AFNetworking iOS"
AF_TVOS_FRAMEWORK_SCHEME="AFNetworking tvOS"
AF_OSX_FRAMEWORK_SCHEME="AFNetworking OS X"

AF_IOS_EXAMPLE_SCHEME="iOS Example"
AF_TVOS_EXAMPLE_SCHEME="tvOS Example"
AF_OSX_EXAMPLE_SCHEME="OS X Example"
```

.env.default:
```
AF_IOS_SDK=iphonesimulator9.1
AF_MAC_SDK=macosx10.11
AF_TVOS_SDK=appletvsimulator9.0

AF_CONFIGURATION=Release

SCAN_WORKSPACE=$AF_WORKSPACE
SCAN_SCHEME=$AF_IOS_FRAMEWORK_SCHEME
SCAN_DESTINATION="OS=9.1,name=iPhone 6s"
SCAN_SDK=$AF_IOS_SDK
SCAN_OUTPUT_DIRECTORY=fastlane/test-output

EXAMPLE_WORKSPACE=$AF_WORKSPACE
EXAMPLE_SCHEME=$AF_IOS_EXAMPLE_SCHEME
EXAMPLE_DESTINATION=$SCAN_DESTINATION
```

.env.ios91
```
SCAN_DESTINATION="OS=9.1,name=iPhone 6s"
EXAMPLE_DESTINATION=$SCAN_DESTINATION
```

Printing out my environment variables, I only see one fastlane environment variable:

```
[10:05:17]: -------------------------------------------------
[10:05:17]: --- Step: Verifying required fastlane version ---
[10:05:17]: -------------------------------------------------
[10:05:17]: fastlane version valid
[10:05:17]: Driving the lane 'test_framework' 🚀
[10:05:17]: rvm_bin_path=/Users/kevin.harwood/.rvm/bin
[SCRUBBED]
[10:05:17]: FASTLANE_LANE_NAME=test_framework
```

**Comment by kcharwood** [comment]
Note I saw travis fail with 1.38.0, ran my lane locally on 1.37.0 and it passed, upgraded locally to 1.38.0, and it failed

**Comment by carloe** [comment]
Same issue here

**Comment by kcharwood** [comment]
Also note there is no log indicating environment variables were loaded:

```
[10:12:08]: -------------------------------------------------
[10:12:08]: --- Step: Verifying required fastlane version ---
[10:12:08]: -------------------------------------------------
[10:12:08]: fastlane version valid
[10:12:08]: Driving the lane 'test_framework' 🚀
```

**Comment by kcharwood** [comment]
@KrauseFx perhaps [this is](https://github.com/fastlane/fastlane/commit/ed951bf30bb1405a312cc2f87aadb30cf013c078) the culprit?

**Comment by iamjason** [comment]
:+1: @kcharwood thanks for pointing this out.  I'm having the same issue.

**Comment by janverhulst** [comment]
Same issue here ...

**Comment by marciomeschini** [comment]
+1

**Comment by kcharwood** [comment]
Addressed in #823
